src/openms/include/OpenMS/ANALYSIS/NUXL/NuXLPresets.h

```diff
@@ -252,7 +252,25 @@ namespace OpenMS
     "A:+C4H6O2-NH3-H3PO4"
   };  
     
-  static constexpr std::array<const char*, 24> modifications_RNA_FA
+  static constexpr std::array<const char*, 12> modifications_RNA_FA
+  {
+    "G:C",
```

Comment:
probably works but usually I add the plus sign
e.g. "G:+C"

Reply:
ok i will update